P2CODE/AA2/Camera/schema.json

```diff
@@ -0,0 +1,138 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
```

Comment:
possibly this model has to be merged with the existing Camera in device subject

Reply:
There are three data models in this PR that have to be extended Camera, Modbus, and UnmannedAerialVehicle. Let me know what is the procedure after approval of the PR.